roosevelt.js

```diff
@@ -44,6 +47,19 @@ module.exports = function(params) {
 
   console.log(('💭  ' + 'Starting ' + app.get('appName') + ' in ' + app.get('env') + ' mode...').bold);
 
+  // check if env comes from cmd line then leave as is, else use params
```

Comment:
once the param is set by the command line flag, you can just move this whole block into sourceParams, where it belongs